src/screens/ProfileScreen/index.js

```diff
@@ -708,6 +708,9 @@ const ProfileScreen = ({route}) => {
               source={SOURCE_MY_PROFILE}
               hideThreeDot={false}
               showAnonymousOption={true}
+              onDeletePost={() => onDeletePost()}
+              isSelf={item.is_self}
+              isShowDelete={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props `onDeletePost`, `isSelf`, and `isShowDelete` have been added to the component. Ensure that these props are properly handled within the component and that they don't introduce any unexpected behavior.
```diff
+ onDeletePost={() => onDeletePost()}
+ isSelf={item.is_self}
+ isShowDelete={true}
  onHeaderOptionClicked={() => onHeaderOptionClicked(item)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->